dotnet/src/SemanticKernel.Skills/Skills.UnitTests/OpenAPI/OpenApiDocumentParserV20Tests.cs

```diff
@@ -61,15 +79,10 @@ public void ItCanParsePutOperationBodyOfV20SpecSuccessfully()
     }
 
     [Fact]
-    public void ItCanParsePutOperationMetadataOfV20SpecSuccessfully()
+    public void ItCanParsePutOperationMetadataOfSuccessfully()
```

Comment:
nit: `MetadataOf` or just `Metadata`? Same in [OpenApiDocumentParserV30Tests.cs](https://github.com/microsoft/semantic-kernel/pull/312/files#diff-6831a1f64f2507111e2fde1e83415dfaccc708e48166ef0c60bfa69140c55616)

Reply:
Fixed.